src/server.c

```diff
@@ -3583,6 +3583,19 @@ int commandCheckArity(client *c, sds *err) {
     return 1;
 }
 
+uint64_t getCommandFlags(client *c) {
+    uint64_t cmd_flags = c->cmd->flags;
+
+    if (c->cmd->proc == fcallCommand || c->cmd->proc == fcallroCommand) {
+        cmd_flags = fcallGetCommandFlags(c, cmd_flags);
+    } else if (c->cmd->proc == evalCommand || c->cmd->proc == evalRoCommand ||
+               c->cmd->proc == evalShaCommand || c->cmd->proc == evalShaRoCommand) {
```

Comment:
styling. multi-line if conditions have the opening brace on the next line. 8-(
```suggestion
               c->cmd->proc == evalShaCommand || c->cmd->proc == evalShaRoCommand)
      {
```